survey/HTML-syntax-in-EPUB.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Survey on the Impact of Allowing HTML Syntax in EPUB
+date: 2025-05-08
+---
+
+The Publishing Maintenance Working Group (PMWG) plans to allow the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax) for the HTML content document format in EPUB 3.4, instead of restricting it to the [XML Syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) as it was done for earlier versions of EPUB 3.
+
+Use of the XML syntax is diminishing outside of publishing. The syntax is no longer being maintained. No new features will be added, and the syntax will not be updated even when related technologies change. Eventually, relying on a deprecated syntax could cause errors and prevent innovation. By introducing HTML syntax into the EPUB package, ebooks can continue to evolve into the future.
+
+The change does not affect the document formats currently allowed in an EPUB publication; they will remain valid as they are today. This change merely removes the restriction to the XML syntax.
+
+PMWG is seeking your valuable feedback. The purpose of this survey is to get the temperature of the community about the technical change. The Working Group hopes to discover potential obstacles so we can make the transition easier. Your insights will help shape the future of EPUB standards. 
```

Comment:
Instead of "hopes to discover potential obstacles" as discussed on the call today, could this be changed to "The Working Group will use your feedback to help identify any obstacles ..."?

```suggestion
PMWG is seeking your valuable feedback. The purpose of this survey is to get the temperature of the community about the technical change. The Working Group will use your feedback to help identify any potential obstacles so we can make the transition easier. Your insights will help shape the future of EPUB standards. 
```

Reply:
Ya, I don't think there's a way to make a pull request that we'll see here off someone's local fork. Suggesting changes through the comments is the only way I know of as github handles committing it back to the fork. It looked like this had been forgotten which is why I suggested a change.

Reply:
I would have preferred to be able to branch the repo here, but I was not authorised.